randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -1,857 +1,882 @@
+
+----------------
+Climb
+Extra - map_name: 796BA
+Extra - asset_id: 26
+> Door to Crateria Super Missile Room; Heals? False; Spawn Point
+  * Layers: default
+  * Power Bomb Door to Crateria Super Missile Room/Door to Climb (South)
+  > Door to Parlor
+      All of the following:
+          Morph Ball
+          Morph Ball Bombs or Power Bombs
+
+> Door to Parlor; Heals? False
+  * Layers: default
+  * Normal Door to Parlor/Door to Climb
+  > Door to Crateria Super Missile Room
+      All of the following:
+          Morph Ball
+          Morph Ball Bombs or Power Bombs
+  > Door to Old Mother Brain
+      Trivial
+  > Door from Crateria Super Missile Room
+      All of the following:
+          Morph Ball
+          Morph Ball Bombs or Power Bombs
+
+> Door to Old Mother Brain; Heals? False
+  * Layers: default
+  * Normal Door to Old Mother Brain/Door to Climb
+  > Door to Parlor
+      Trivial
+
+> Door from Tourian Escape Room 4; Heals? False
+  * Layers: default
+  * Permanently Locked to Tourian Escape Room 4/Door to Climb
+  > Door to Parlor
+      Trivial
+  > Door to Old Mother Brain
+      Trivial
+
+> Door from Crateria Super Missile Room; Heals? False
+  * Layers: default
+  * Permanently Locked to Crateria Super Missile Room/Door to Climb (North)
+  > Door to Parlor
+      All of the following:
+          Morph Ball
+          Morph Ball Bombs or Power Bombs
+
+----------------
+Crateria Super Missile Room
+Extra - map_name: 799F9
+Extra - asset_id: 27
+> Door to Climb (South); Heals? False; Spawn Point
+  * Layers: default
+  * Normal Door to Climb/Door to Crateria Super Missile Room
+  > Pickup (Super Missile Expansion)
+      All of the following:
+          Speed Booster and Shinespark (Beginner)
+          Ice Beam or Shinespark (Advanced)
+
+> Pickup (Super Missile Expansion); Heals? False
+  * Layers: default
+  * Pickup 11; Major Location? False
+  > Door to Climb (North)
+      All of the following:
+          Morph Ball
+          Grapple Beam or Space Jump
+
+> Door to Climb (North); Heals? False
+  * Layers: default
+  * Normal Door to Climb/Door from Crateria Super Missile Room
+  > Pickup (Super Missile Expansion)
+      All of the following:
+          Morph Ball
+          Grapple Beam or Space Jump
+
+----------------
+Old Mother Brain
+Extra - map_name: 7975C
+Extra - asset_id: 28
+> Door to Climb; Heals? False; Spawn Point
+  * Layers: default
+  * Normal Door to Climb/Door to Old Mother Brain
+  > Door to Elevator to Blue Brinstar
+      Trivial
+  > Pickup (Missile Expansion)
+      All of the following:
+          After Crateria Wakeup
+          Any of the following:
+              Screw Attack
+              All of the following:
+                  Morph Ball
+                  Morph Ball Bombs or Power Bombs
+              Speed Booster and Blue Suit Glitch (Intermediate) and Short Charge (Advanced)

```

Comment:
I remember vaguely seeing Oats & Goats do this without blue suit glitch (but using morph ball). I'll find a video in a bit.